cmake/FindCharm.cmake

```diff
@@ -53,6 +53,17 @@ find_library(CHARM_LIBCK
     PATH_SUFFIXES lib
     HINTS ${CHARM_ROOT} ENV CHARM_ROOT
 )
+find_library(CHARM_EVERYLB
+    NAMES libmoduleEveryLB.a
+    PATH_SUFFIXES lib64 lib
+    HINTS ${CHARM_ROOT} ENV CHARM_ROOT
+)
+
+if("${CHARM_EVERYLB}" STREQUAL "CHARM_EVERYLB-NOTFOUND")
+  message(SEND_ERROR "Could not find charm module EveryLB.\
```

Comment:
Should this be `FATAL_ERROR` to abort cmake at this point, or did you choose `SEND_ERROR` on purpose?

(optional) You don't need the `\` line continuation here, `message` supports syntax like this:

```cmake
message(FATAL_ERROR "Could not find charm module EveryLB. "
  "Make sure you have built the LIBS target when building charm++.")
```

Reply:
I chose `SEND_ERROR` because I felt it made sense to error and proceed to process other cmake problems if any. I've squashed the line continuation change.